platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiActivitiesInSpaceTestIT.java

```diff
@@ -21,7 +21,7 @@
 
 @Tag("smoke")
```

Comment:
I think changing it to "sniff" and executing the test with -Dselenide.test.tags.include=sniff should be enought, no ?